README.md

```diff
@@ -40,6 +41,9 @@ The following targets have been tested and work with these examples:
 	* DISCO_L475VG_IOT01A (ref B-L475E-IOT01A)
 
 	<span> **Important:** if an ST shield is used with the K64F board, an hardware is patch required. Check out https://developer.mbed.org/teams/ST/code/X_NUCLEO_IDB0XA1/ for more information.</span>
+    
+The following board is no longer supported:
```

Comment:
That's a bit alarming; please reword to indicate that it doesn't work with new API ATM but will in a latter release.